### PR TITLE
Changes to check-uk-visa for Estonia, Latvia, Taiwan, UKOT and DATV

### PR DESCRIPTION
**Pull request by leenagupte**
Trello card: https://trello.com/c/ZulNTOEn
Google doc for reference: https://docs.google.com/document/d/1vYQqcPIhB9Q3F28nlgT-9sLdjucMsMWbJbfB_VMuNSM/edit

## Factcheck
[check-uk-visa](https://smart-answers-pr-2973.herokuapp.com/check-uk-visa/y/estonia)

## Expected Changes
* [URL on GOV.UK](https://www.gov.uk/check-uk-visa/y/estonia)
   * Insert a question into the flow for Estonia and Latvia
   * Add new outcomes for users coming to the UK to get married 
   * Fix content for some Taiwan in transit outcomes
   * Content changes to other existing outcomes

### Before
screen shot 2017-03-23 at 15 14 54

### After
screen shot 2017-03-23 at 15 15 17

### CODE REVIEW

**Comment by ikennaokpala** [comment]
@leenagupte I have created this PR #2979  help explain and resolve the discrepancy in b02c4e7.
If you can kindly review merged and rebase this..  This PR would become more clearer.

**Comment by leenagupte** [comment]
@ikennaokpala I have rebased against and picked up the changes in PR #2979.

File: lib/smart_answer_flows/check-uk-visa/outcomes/outcome_marriage_visa_nat_datv.govspeak.erb [hunk 1]
```diff
@@ -5,6 +5,10 @@
 <% content_for :body do %>
   <%= render partial: 'stateless_or_refugee.govspeak.erb', locals: {calculator: calculator} %>
 
+  <% if calculator.passport_country_is_estonia? || calculator.passport_country_is_latvia? %>
```
Comment (leenagupte) [inline lib/smart_answer_flows/check-uk-visa/outcomes/outcome_marriage_visa_nat_datv.govspeak.erb]: @ikennaokpala I see your point. I would have made sense to split the logic change in the flow from the additional logic in the template.

**Comment by ikennaokpala** [comment]
Rebased this PR from c7633f9 to

**Pull request merged**